WcaOnRails/app/models/website_contact.rb

```diff
@@ -24,6 +24,8 @@ def to_email
       if competition.present?
         return ValidateEmail.valid?(competition.contact) ? competition.contact : competition.managers.map(&:email)
       end
+    elsif inquiry == "results_team"
+      return "results@worldcubeassociation.org"
```

Comment:
Please try to use proper data bindings wherever possible: `Team.wrt.email`

(I know that `contact@` is hardcoded below, but that's because it's a general contact address and not WCT's _actual_ email)

Reply:
Okay, done.